iml-manager-cli/src/error.rs

```diff
@@ -66,6 +66,7 @@ pub enum ImlManagerCliError {
     IoError(std::io::Error),
     CombineEasyError(combine::stream::easy::Errors<char, &'static str, usize>),
     ReqwestError(reqwest::Error),
+    ProfileNotFound(String),
```

Comment:
I think we should probably not tack on new Error enum variants for things like this. 

It adds more complexity where we end up just printing all the errors in the same way.

This could instead be a bail out when the profile isn't found or just a plain `IoError`.

Reply:
Sure, I can remove it. My intention here was to simplify the call and remove the match expression. For example:

```
let profile = match profile_opt {
                Some(p) => p,
                None => {
                    display_error(format!("Unknown profile {}", config.profile));

                    return Ok(());
                }
            };
```
became:
```
let profile =
        get_profile(&config, profiles).ok_or(ImlManagerCliError::ProfileNotFound(profile_name))?;
```

Reply:
Understood. `Io::Error` provides the same thing without having to create a new variant.

Reply:
I updated to:
```
let profile = get_profile(&config, profiles).ok_or(Error::new(
        ErrorKind::NotFound,
        format!("{} profile not found.", profile_name),
    ))?;
```
I'm in the process of testing the CLI on my cluster.